pkg/controllers/management/auth/crtb_handler.go

```diff
@@ -205,7 +205,7 @@ func (c *crtbLifecycle) reconcileLabels(binding *v3.ClusterRoleTemplateBinding)
 	    2. CRTB.UID is label key for the RB, CRTB.UID=crtb-in-project-binding-owner (in the namespace of each project in the cluster that the user has access to)
 	Using above labels, list the CRB and RB and update them to add a label with ns+name of CRTB
 	*/
-	if binding.Labels[rtbCrbRbLabelsUpdated] == "true" {
+	if binding.Labels != nil && binding.Labels[rtbCrbRbLabelsUpdated] == "true" {
```

Comment:
These nil checks are redundant, you will get the zero value from the map trying to access the nil map - see https://play.golang.org/p/1nGFExa4DwJ

Reply:
oh thanks!